### PR TITLE
fairwinds-115-conformanceresults

### PR DESCRIPTION
**Pull request by bitbucket90**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [X ] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [N/A ] Did you include the product/project logo in SVG, EPS or AI format?  
* [N/A ] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [ ] If your product/project is open source, did you include the `repo_url`?
* [ X] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

We have no logo for this offering (yet). It is simply text

### CODE REVIEW

**Comment by taylorwaggoner** [comment]
@bitbucket90 thanks for submitting! There are a few things we need to resolve before I can certify you.

First, please make sure to add a link to your logo in SVG, EPS, or AI in the product.yaml file, [per these guidelines.  ](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#productyaml).  Please also add the product type and product description.  

Please also copy and paste the installation and configuration instructions into your README.md file, [per these guidelines](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions).

Let me know if you have any questions! Thanks!

**Comment by bitbucket90** [comment]
Added. Thank you @taylorwaggoner

**Comment by taylorwaggoner** [comment]
@bitbucket90 , thanks!  Are there more detailed directions on how to install and configure your Kubernetes implementation?  Such as how to create/deploy clusters?

**Comment by bitbucket90** [comment]
> @bitbucket90 , thanks! Are there more detailed directions on how to install and configure your Kubernetes implementation? Such as how to create/deploy clusters?

We deploy these clusters for the customer (based on our own and industry best practices). We do this in Terraform/KOPS. But not sure that fits well into that readme.

**Comment by taylorwaggoner** [comment]
@bitbucket90, would you be able to paste in a summary of the installation steps? In particular, is there any special configuration necessary to get the tests to pass? We're trying to provide guidance so that your customers can later validate your conformance if they wish to.  Thanks!

**Comment by bitbucket90** [comment]
Updated - we run this as close to vanilla kubernetes as possible thus the customer should not be required to change anything. While we handle the install I have included a very brief summary as the code base can change based on various deviations. The conformance testing is becoming part of our RnD process also.

**Comment by taylorwaggoner** [comment]
@bitbucket90 - We'd also like to see the instructions for how to provision a cluster.  Basically, what would someone need to do from start to finish to get the same test results.  Thanks!

**Comment by bitbucket90** [comment]
> @bitbucket90 - We'd also like to see the instructions for how to provision a cluster. Basically, what would someone need to do from start to finish to get the same test results. Thanks!

Each one of our deployed infrastructures for the customer is custom built and we do rigorous information gathering from the start, each of these are built around open source software we offer and in house tooling. There for each customer is given a managed cluster (or a few), thus there are no installation instructions just a deployed managed cluster that we build - We do these conformance tests in our build environment each end user would have access to the cluster and be able to run this test. The provided documentation is about as elaborate as I could pull together as each customer is different. I discussed this with a few engineers and our CTO. @taylorwaggoner

File: v1.15/fairwinds/product.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+vendor: Fairwinds Ops
+name: ClusterOps
+version: 1.15.7
+website_url: https://www.fairwinds.com
+documentation_url: https://www.fairwinds.com/clusterops
+product_logo_url: https://www.fairwinds.com/hubfs/FW-Logo-Warm-Navy.svg
+type: hosted
+description: Manged Kubernetes platform hosted on AWS/GCP/AZURE
```
Comment (dankohn) [inline v1.15/fairwinds/product.yml:8]: Managed

**Comment by dankohn** [comment]
Could you please just add a step like:

1. Arrange for a standard cluster to be installed for you using the default configuration.

And then give the curl command or Sonobuoy steps (or both) for a customer to get the same results.

File: v1.15/fairwinds/product.yml (hunk 1 above)
Comment (ejether) [reply v1.15/fairwinds/product.yml]: Fixed.

**Comment by ejether** [comment]
@dankohn @taylorwaggoner  I'll be working with you on the rest of this conformance PR. I've incorporated your feedback as I understood.  Please let me know if you'd like any other changes.

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes
https://landscape.cncf.io/format=card-mode&organization=fairwinds&selected=fairwinds-cluster-ops

**Comment by ejether** [comment]
Thanks!

**Pull request merged**